.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,44 @@
+<!--  Thanks for sending a pull request!  Here are some tips for you:
+
+1. If this is your first time, please read our contributor guidelines: https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/CONTRIBUTING.md and developer guide https://github.com/vmware-tanzu/carvel-kapp-controller/blob/develop/docs/dev.md
+-->
+
+#### What type of PR is this?
+
+<!--
+Please add a label to your pull request that matches the type of change you are making.
```

Comment:
I think we'll want to change this wording since non-maintainers can't add labels ([GH docs](https://docs.github.com/en/issues/using-labels-and-milestones-to-track-work/managing-labels#applying-a-label)).